Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -169,6 +169,7 @@ class TextureCacheBase
 
   TCacheEntry* Load(const u32 stage);
   static void InvalidateAllBindPoints() { valid_bind_points.reset(); }
+  static void InvalidateBindPoint(u8 stage) { valid_bind_points.reset(stage); }
```

Comment:
Every place this is currently used is implicitly truncating to `u8`, which can cause warnings.